data/json/requirements/cooking_components.json

```diff
@@ -77,7 +77,15 @@
         [ "egg_goose_canadian", 1 ],
         [ "egg_turkey", 1 ],
         [ "egg_pheasant", 1 ],
-        [ "egg_cockatrice", 1 ]
+        [ "egg_cockatrice", 1 ],
+        [ "egg_goose", 1 ],
+        [ "egg_goose_golden", 1 ],
+        [ "egg_hummingbird", 1 ],
```

Comment:
Apparently hummingbird eggs are ridiculously tiny irl so the ratio is not 1:1 with the other eggs. 


https://pbs.twimg.com/media/Cir_3u0WYAArUgj.jpg

Reply:
That is true, but I had originally checked their copy-froms and they end up copying from crows (which copy from chickens). However your comment made me double check and the hummingbirds in-game are not some enlarged mutant variety, but just normal-sized hummingbirds - so copying from chickens is definitely wrong. I will put in another commit to resolve this when I can.

Reply:
I decided to simply change their requirement to the `baby_count` for the two biggest offenders, so one portion of eggs for cooking is one litter. There are some bigger issues with the existing (not ported from tropiclysm) bird eggs, but I think that's a little out of scope.